lib/tasks/rspec.rake

```diff
@@ -7,6 +7,9 @@ namespace :knapsack_pro do
 
   desc "Generate JSON report for test suite based on default test pattern or based on defined pattern with ENV vars"
   task :rspec_test_example_detector do
+    # ignore the `SPEC_OPTS` options to not affect RSpec execution within this rake task
+    ENV.delete('SPEC_OPTS')
```

Comment:
This is run in a shelled out process so it doesn't affect the ENV for other processes?

Reply:
correct